monitor/io_stat_provider.cpp

```diff
@@ -76,7 +76,8 @@ void dump_io_pool_stats(struct dnet_io_pool &io_pool,
 
 	rapidjson::Value nonblocking(rapidjson::kObjectType);
 	nonblocking.AddMember("current_size", dnet_get_pool_queue_size(io_pool.recv_pool_nb.pool), allocator);
-	value.AddMember("nonblocking", nonblocking, allocator);
+	const auto mode = io_pool.recv_pool_nb.pool->mode == DNET_WORK_IO_MODE_LIFO ? "lifo" : "nonblocking";
+	value.AddMember(mode, nonblocking, allocator);
```

Comment:
To be consistent:
```suggestion
	value.AddMember(mode, pool_info, allocator);
```
Or maybe I'm not understand: why "lifo" and "nonblocking" are put here as alternatives? Maybe let
`io_pool.recv_pool_nb.pool->mode` to be `"lifo"` and `"fifo"`? I think code readers can get confused.

The same question is about all the other changes.

Reply:
'nonblocking' is current name and I left it for backward compatibility.